kernel/common/get_slice.rs

```diff
@@ -1,8 +1,6 @@
-use core::slice;
-use core::str;
 use collections::range::RangeArgument;
 use core::ops::Range;
-use core::cmp::{max, min};
+use core::cmp::min;
```

Comment:
No they are in the kernel but apparently they are still in the orbital branch:
https://github.com/redox-os/redox/blob/orbital/kernel/common/slice/get_slice.rs